ldes-server-infra-mongo/mongock-changeset-1/README.md

```diff
@@ -0,0 +1,9 @@
+# Mongock Changeset 1
```

Comment:
Maybe we should add some documentation to our changesets as well (kind of like release notes?):
- Provides clarity about the purpose of the migration and the changes being made to the database schema.
- Have visibility into the changes being made to the database schema. This can help identify potential issues before they occur.